<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferDialog</class>
 <widget class="QDialog" name="TransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>664</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="9" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="pathGroup" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="textDest"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Source:</string>
        </property>
        <property name="buddy">
         <cstring>textSource</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="buttonSourceFile">
        <property name="toolTip">
         <string>Choose file</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border: 0; }</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Destination:</string>
        </property>
        <property name="buddy">
         <cstring>textDest</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="buttonSourceFolder">
        <property name="toolTip">
         <string>Choose folder</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border: 0; }</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="buttonDest">
        <property name="toolTip">
         <string>Choose folder</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border: 0; }</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="textSource"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="modeGroup">
         <property name="title">
          <string>Mode</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbCopy">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Copy</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbMove">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Move</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rbSync">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sync</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbSyncDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Delete during transfer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Delete after transfering</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Delete before transfering</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="skipFilesGroup">
         <property name="title">
          <string>Skip files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="3" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2" alignment="Qt::AlignVCenter">
           <widget class="QComboBox" name="cbCompare">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentText">
             <string>Size &amp; mod-time</string>
            </property>
            <item>
             <property name="text">
              <string>Size &amp; mod-time</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Size &amp; checksum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Only mod-time</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Only size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Only checksum</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="checkSkipNewer">
            <property name="text">
             <string>Skip files that are newer on the destination</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" alignment="Qt::AlignBottom">
           <widget class="QCheckBox" name="checkCompare">
            <property name="toolTip">
             <string>-ignore-times</string>
            </property>
            <property name="text">
             <string>Compare</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QCheckBox" name="checkSkipExisting">
            <property name="text">
             <string>Skip all files that exist</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkVerbose">
            <property name="text">
             <string>Verbose output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkSameFilesystem">
            <property name="text">
             <string>Don't cross filesystem boundaries</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkDontUpdateModified">
            <property name="text">
             <string>Don't update mod-time if files are identical</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Extra arguments:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textExtra"/>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Task description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textDescription"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>Transfer</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Checkers</string>
         </property>
         <property name="buddy">
          <cstring>spinCheckers</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinCheckers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Bandwidth (KiB/s or b|k|M|G suffix)</string>
         </property>
         <property name="buddy">
          <cstring>textBandwidth</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="textBandwidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Minimum size (KiB or b|k|M|G suffix)</string>
         </property>
         <property name="buddy">
          <cstring>textMinSize</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="textMinSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Minimum age (s or ms|s|m|h|d|w|M|y suffix)</string>
         </property>
         <property name="buddy">
          <cstring>textMinAge</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="textMinAge">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum age (s or ms|s|m|h|d|w|M|y suffix)</string>
         </property>
         <property name="buddy">
          <cstring>textMaxAge</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="textMaxAge">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Maximum depth</string>
         </property>
         <property name="buddy">
          <cstring>spinMaxDepth</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinMaxDepth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Connect timeout</string>
         </property>
         <property name="buddy">
          <cstring>spinConnectTimeout</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="spinConnectTimeout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>IO idle timeout</string>
         </property>
         <property name="buddy">
          <cstring>spinIdleTimeout</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="spinIdleTimeout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Retries</string>
         </property>
         <property name="buddy">
          <cstring>spinRetries</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSpinBox" name="spinRetries">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Low level retries</string>
         </property>
         <property name="buddy">
          <cstring>spinLowLevelRetries</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="spinLowLevelRetries">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinTransfers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Transfers</string>
         </property>
         <property name="buddy">
          <cstring>spinTransfers</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab4">
      <attribute name="title">
       <string>Exclude</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkDeleteExcluded">
         <property name="text">
          <string>Delete excluded files from destination</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textExclude">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textSource</tabstop>
  <tabstop>buttonSourceFile</tabstop>
  <tabstop>buttonSourceFolder</tabstop>
  <tabstop>textDest</tabstop>
  <tabstop>buttonDest</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>rbCopy</tabstop>
  <tabstop>rbMove</tabstop>
  <tabstop>rbSync</tabstop>
  <tabstop>cbSyncDelete</tabstop>
  <tabstop>checkSkipNewer</tabstop>
  <tabstop>checkSkipExisting</tabstop>
  <tabstop>checkCompare</tabstop>
  <tabstop>cbCompare</tabstop>
  <tabstop>checkVerbose</tabstop>
  <tabstop>checkSameFilesystem</tabstop>
  <tabstop>checkDontUpdateModified</tabstop>
  <tabstop>spinTransfers</tabstop>
  <tabstop>spinCheckers</tabstop>
  <tabstop>textBandwidth</tabstop>
  <tabstop>textMinSize</tabstop>
  <tabstop>textMinAge</tabstop>
  <tabstop>textMaxAge</tabstop>
  <tabstop>spinMaxDepth</tabstop>
  <tabstop>spinConnectTimeout</tabstop>
  <tabstop>spinIdleTimeout</tabstop>
  <tabstop>spinRetries</tabstop>
  <tabstop>spinLowLevelRetries</tabstop>
  <tabstop>checkDeleteExcluded</tabstop>
  <tabstop>textExclude</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rbSync</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbSyncDelete</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCompare</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbCompare</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
